### ISSUE TITLE
0.10.12 breaks my nuget server

### ISSUE THREAD

**Issue by helge000** [chocolatey/choco#1775]
### What You Are Seeing?
After upgrading to v 0.10.12 (or .13 for that matter) choco fails to connect to my nuget server and skips installing the package altogether (exit 1)

I am using [simple-nuget-server](https://github.com/Daniel15/simple-nuget-server)

### What is Expected?
- Package installed, or at least broken source skipped

### How Did You Get This To Happen? (Steps to Reproduce)
- Install any package

### Workaround
```
choco install --version=0.10.11 chocolatey --force
```

### Output Log
```
C:\>choco install --debug nextcloud-client
Chocolatey v0.10.13
Chocolatey is running on Windows v 10.0.16299.0
Command line: "C:\ProgramData\chocolatey\choco.exe" install --debug nextcloud-client
Received arguments: install --debug nextcloud-client
Performing validation checks.
Global Configuration Validation Checks:
 - Package Exit Code / Exit On Reboot = Checked
System State Validation Checks:
 Reboot Requirement Checks:
 - Pending Computer Rename = Checked
 - Pending Component Based Servicing = Checked
 - Pending Windows Auto Update = Checked
 - Pending File Rename Operations = Flagged
2 validations performed. 1 success(es), 1 warning(s), and 0 error(s).

Validation Warnings:
 - A pending system reboot request has been detected, however, this is
   being ignored due to the current Chocolatey configuration.  If you
   want to halt when this occurs, then either set the global feature
   using:
     choco feature enable -name=exitOnRebootDetected
   or pass the option --exit-when-reboot-detected.

The source 'https://pkg.m-box.de/;https://chocolatey.org/api/v2/' evaluated to a 'normal' source type

NOTE: Hiding sensitive configuration data! Please double and triple
 check to be sure no sensitive data is shown, especially if copying
 output to a gist for review.
Configuration: CommandName='install'|
CacheLocation='C:\Users\autombox\AppData\Local\Temp\chocolatey'|
ContainsLegacyPackageInstalls='True'|
CommandExecutionTimeoutSeconds='2700'|WebRequestTimeoutSeconds='30'|
Sources='https://pkg.m-box.de/;https://chocolatey.org/api/v2/'|
SourceType='normal'|Debug='True'|Verbose='False'|Trace='False'|
Force='False'|Noop='False'|HelpRequested='False'|
UnsuccessfulParsing='False'|RegularOutput='True'|QuietOutput='False'|
PromptForConfirmation='True'|AcceptLicense='False'|
AllowUnofficialBuild='False'|Input='nextcloud-client'|
AllVersions='False'|SkipPackageInstallProvider='False'|
PackageNames='nextcloud-client'|Prerelease='False'|ForceX86='False'|
OverrideArguments='False'|NotSilent='False'|
ApplyPackageParametersToDependencies='False'|
ApplyInstallArgumentsToDependencies='False'|IgnoreDependencies='False'|
AllowMultipleVersions='False'|AllowDowngrade='False'|
ForceDependencies='False'|Information.PlatformType='Windows'|
Information.PlatformVersion='10.0.16299.0'|
Information.PlatformName='Windows 10'|
Information.ChocolateyVersion='0.10.13.0'|
Information.ChocolateyProductVersion='0.10.13'|
Information.FullName='choco, Version=0.10.13.0, Culture=neutral, PublicKeyToken=79d02ea9cad655eb'|

Information.Is64BitOperatingSystem='True'|
Information.Is64BitProcess='True'|Information.IsInteractive='True'|
Information.UserName='autombox'|Information.UserDomainName='V-DP-04'|
Information.IsUserAdministrator='True'|
Information.IsUserSystemAccount='False'|
Information.IsUserRemoteDesktop='False'|
Information.IsUserRemote='False'|Information.IsProcessElevated='True'|
Information.IsLicensedVersion='False'|Information.LicenseType='Foss'|
Features.AutoUninstaller='True'|Features.ChecksumFiles='True'|
Features.AllowEmptyChecksums='False'|
Features.AllowEmptyChecksumsSecure='True'|
Features.FailOnAutoUninstaller='False'|
Features.FailOnStandardError='False'|Features.UsePowerShellHost='True'|
Features.LogEnvironmentValues='False'|Features.LogWithoutColor='False'|
Features.VirusCheck='False'|
Features.FailOnInvalidOrMissingLicense='False'|
Features.IgnoreInvalidOptionsSwitches='True'|
Features.UsePackageExitCodes='True'|
Features.UseEnhancedExitCodes='True'|
Features.UseFipsCompliantChecksums='False'|
Features.ShowNonElevatedWarnings='True'|
Features.ShowDownloadProgress='True'|
Features.StopOnFirstPackageFailure='False'|
Features.UseRememberedArgumentsForUpgrades='False'|
Features.IgnoreUnfoundPackagesOnUpgradeOutdated='False'|
Features.SkipPackageUpgradesWhenNotInstalled='False'|
Features.RemovePackageInformationOnUninstall='False'|
Features.ExitOnRebootDetected='False'|
Features.LogValidationResultsOnWarnings='True'|
Features.ScriptsCheckLastExitCode='False'|
ListCommand.LocalOnly='False'|
ListCommand.IdOnly='False'|ListCommand.IncludeRegistryPrograms='False'|
ListCommand.PageSize='25'|ListCommand.Exact='False'|
ListCommand.ByIdOnly='False'|ListCommand.ByTagOnly='False'|
ListCommand.IdStartsWith='False'|ListCommand.OrderByPopularity='False'|
ListCommand.ApprovedOnly='False'|
ListCommand.DownloadCacheAvailable='False'|
ListCommand.NotBroken='False'|
ListCommand.IncludeVersionOverrides='False'|
UpgradeCommand.FailOnUnfound='False'|
UpgradeCommand.FailOnNotInstalled='False'|
UpgradeCommand.NotifyOnlyAvailableUpgrades='False'|
UpgradeCommand.ExcludePrerelease='False'|
NewCommand.AutomaticPackage='False'|
NewCommand.UseOriginalTemplate='False'|SourceCommand.Command='unknown'|
SourceCommand.Priority='0'|SourceCommand.BypassProxy='False'|
SourceCommand.AllowSelfService='False'|
SourceCommand.VisibleToAdminsOnly='False'|
FeatureCommand.Command='unknown'|ConfigCommand.Command='unknown'|
ApiKeyCommand.Remove='False'|PinCommand.Command='unknown'|
OutdatedCommand.IgnorePinned='False'|
Proxy.Location='******'|
Proxy.BypassList='pkg.m-box.de'|Proxy.BypassOnLocal='True'|
_ Chocolatey:ChocolateyInstallCommand - Normal Run Mode _
Installing the following packages:
nextcloud-client
By installing you accept licenses for the packages.
Using proxy server '******'.
Proxy has a bypass list of 'pkg.m-box.de'.
Using 'https://pkg.m-box.de/'.
- Supports prereleases? 'True'.
- Is ServiceBased? 'True'.

Chocolatey installed 0/0 packages.
 See the log for details (C:\ProgramData\chocolatey\logs\chocolatey.log).
Chocolatey had an error occur:
System.InvalidOperationException: Could not connect to the feed specified at 'https://pkg.m-box.de/'. Please verify that the package source (located in the Package Manager Settings) is valid and ensure your network connectivity. ---> System.Data.Services.Client.DataServiceQueryException: An error occurred while processing this request. ---> System.Data.Services.Client.DataServiceClientException: Unknown filter &quot;(tolower(Id) eq 'nextcloud-client') and IsLatestVersion&quot;
   at System.Data.Services.Client.QueryResult.Execute()
   at System.Data.Services.Client.DataServiceRequest.Execute[TElement](DataServiceContext context, QueryComponents queryComponents)
   --- End of inner exception stack trace ---
   at System.Data.Services.Client.DataServiceRequest.Execute[TElement](DataServiceContext context, QueryComponents queryComponents)
   at NuGet.DataServiceQueryWrapper`1.Execute[TResult](Func`1 action)
   --- End of inner exception stack trace ---
   at NuGet.DataServiceQueryWrapper`1.Execute[TResult](Func`1 action)
   at NuGet.DataServiceQueryWrapper`1.<GetAll>d__14.MoveNext()
   at System.Collections.Generic.List`1..ctor(IEnumerable`1 collection)
   at System.Linq.Enumerable.ToList[TSource](IEnumerable`1 source)
   at chocolatey.infrastructure.app.services.NugetService.find_package(String packageName, SemanticVersion version, ChocolateyConfiguration config, IPackageRepository repository)
   at chocolatey.infrastructure.app.services.NugetService.install_run(ChocolateyConfiguration config, Action`1 continueAction)
   at chocolatey.infrastructure.app.services.ChocolateyPackageService.perform_source_runner_function[T](ChocolateyConfiguration config, Func`2 function)
   at chocolatey.infrastructure.app.services.ChocolateyPackageService.install_run(ChocolateyConfiguration config)
   at chocolatey.infrastructure.app.runners.GenericRunner.run(ChocolateyConfiguration config, Container container, Boolean isConsole, Action`1 parseArgs)
   at chocolatey.infrastructure.app.runners.ConsoleApplication.run(String[] args, ChocolateyConfiguration config, Container container)
   at chocolatey.console.Program.Main(String[] args)
Exiting with 1
```

**Comment by ferventcoder** [comment]
Possibly related to #1774 and #1397. 

The tl;dr - install the latest beta and look at #1770 on how to make it work if the other aspects do not make it work immediately.

**Comment by ferventcoder** [comment]
Duplicate of #1770

**Comment by ferventcoder** [comment]
@helge000 out of curiosity, what version of NuGet Server do you have implemented?

**Comment by ferventcoder** [comment]
This is not known to be an issue with newer NuGet Servers, at least not the Chocolatey Simple Server implementation of NuGet Server.

**Comment by ferventcoder** [comment]
I took a look, I thought we were talking about NuGet.Server, and this is a PHP product.

**Comment by helge000** [comment]
@ferventcoder - true, it implements nuget v2 by some [nginx rewrites](https://github.com/Daniel15/simple-nuget-server/blob/master/nginx.conf.example). However, this is simple and works quite well for us for some time (we are not using it exclusively for choco)

**Comment by helge000** [comment]
Right, I'll wait for .14-beta, try again and report back to #1770 . In the mean time I'll have to figure out how to integrate this feature with .11; where it is not available (choco is managed by the puppet module)

**Comment by ferventcoder** [comment]
@helge000 for you, install the beta and turn off the optimizations and you will see it continue working well.

**Comment by AdmiringWorm** [comment]
@helge000 a beta for 0.10.14 is already available
https://chocolatey.org/packages/chocolatey/0.10.14-beta-20190321

**Comment by helge000** [comment]
@AdmiringWorm - thanks for the heads up, I just tested it and the fix in #1770 works for me and Simple Nuget Server
